lib/rouge/lexers/docker.rb

```diff
@@ -20,6 +20,10 @@ class Docker < RegexLexer
       state :root do
         rule %r/\s+/, Text
 
+        rule %r/^(FROM)(\s+)(.*)(\s+)(AS)(\s+)(.*)/io do
```

Comment:
**suggestion:** Would it be possible to add an example of this in the `samples/docker` file 🙏🏼 ?

Reply:
Of course, I've added it just now.